osquery/main/main.cpp

```diff
@@ -140,6 +140,7 @@ int startShell(osquery::Initializer& runner, int argc, char* argv[]) {
     retcode = osquery::launchIntoShell(argc, argv);
     // Finally shutdown.
     runner.requestShutdown();
+    runner.waitForShutdown();
```

Comment:
Do you actually need this here? because I `requestShutdown()` already calls `waitShutdown()`. I actually think `requestShutdown()` shouldn't do it because of semantics but just raising the point.